lib/notion/getAllPosts.js

```diff
@@ -37,11 +40,14 @@ export async function getAllPosts ({ includePages = false }) {
       const id = pageIds[i]
       const properties = (await getPageProperties(id, block, schema)) || null
 
-      // Add fullwidth, createdtime to properties
-      properties.createdTime = new Date(
-        block[id].value?.created_time
-      ).toString()
+      // Add fullwidth to properties
       properties.fullWidth = block[id].value?.format?.page_full_width ?? false
+      // Convert date(with its timezone) to unix miliseconds timestamp
+      properties.date = (
+        properties.date?.start_date ?
+          dayjs.tz(properties.date?.start_date, BLOG.timezone) :
```

Comment:
Since we don’t use multiple timezones at the same, I guess running `dayjs.tz.setDefault()` somewhere would be more convenient.